src/exercise/01.js

```diff
@@ -3,12 +3,13 @@
 
 import * as React from 'react'
 
-function Greeting() {
+function Greeting({initialName}) {
   // 💣 delete this variable declaration and replace it with a React.useState call
-  const name = ''
+  const [name, setName] = React.useState(initialName);
 
   function handleChange(event) {
     // 🐨 update the name here based on event.target.value
+    setName(event.target.value);
   }
 
   return (
```

Comment:
I _think_ you'll also want to add `value={name}` to the `input` here :)

Reply:
Hm that's kinda weird because it works without it!?

Reply:
Really?!? Well that's good to know! Thank you!

Nothing makes sense in this world LOL 🤦